.circleci/config.yml

```diff
@@ -240,6 +240,44 @@ jobs:
           command: |
               cd driverkit
               make cleanup_s3
+  "test-infra/scan/terraform":
+    docker:
+      - image: accurics/terrascan:latest
+    steps:
+      - checkout
+      - run:
+          name: Scan terraform code
+          command: |
+            #terrascan scan -d ./config/clusters -i terraform -t aws
```

Comment:
Why is this commented?

Reply:
It remained as a placeholder as it didn't work. We can remove it and add it in a separate PR. WDYT?

Reply:
I fixed it, let's see if it works now. Otherwise we can remove it for now I think and open a separate PR for it

Reply:
Ok, it works. It now prevents to deploy "unsecure" infrastructure code